code/messaging/src/main/java/com/adobe/marketing/mobile/aepcomposeui/uimodels/AepImage.kt

```diff
@@ -16,23 +16,8 @@ package com.adobe.marketing.mobile.aepcomposeui.uimodels
  *
  * @property url The URL of the image.
  * @property darkUrl The URL of the image for dark mode.
- * @property bundle The resource bundle for the image.
- * @property darkBundle The resource bundle for the image in dark mode.
- * @property icon The icon name or identifier.
- * @property iconSize The size of the icon.
- * @property iconColor The color of the icon.
- * @property alt The alt text for the image.
- * @property placeholder The placeholder image for the image.
  */
 data class AepImage(
-    val url: String? = null,
+    val url: String,
```

Comment:
Why did we change this to non null? As per spec its optional
https://wiki.corp.adobe.com/display/adms/AEPImage

Reply:
Done